src/main/java/com/squareup/kotlinpoet/AnnotationSpec.kt

```diff
@@ -26,7 +26,10 @@ import javax.lang.model.util.SimpleAnnotationValueVisitor7
 import kotlin.reflect.KClass
 
 /** A generated annotation on a declaration.  */
-class AnnotationSpec private constructor(builder: AnnotationSpec.Builder) {
+class AnnotationSpec private constructor(
+    builder: AnnotationSpec.Builder,
```

Comment:
nit: 2 space indent, same for all spec classes

Reply:
386254c

Worth mentioning though that my IDE doesn't see a difference between primary constructor param indents and continuation indent (using the square intelliJ code style too)